solr/solr-ref-guide/src/the-terms-component.adoc

```diff
@@ -344,16 +344,4 @@ Result:
 
 == Distributed Search Support
```

Comment:
I think you can just remove this heading and the sentence you added. All of Solr's Search Components support distributed search.

Reply:
Ok